.vscode/launch.json

```diff
@@ -13,6 +13,23 @@
       "request": "attach",
       "processId": "${command:PickProcess}"
     },
+    {
+      "name": "Debug Current Vitest File",
```

Comment:
Code tells me the "Debug Current Vitest File" scope is missing a "request" property.

Reply:
thanks, I've added it